src/core/stdcpp/string.d

```diff
@@ -727,6 +733,8 @@ extern(D):
         }
         else
         {
+            pragma(msg, "libstdc++ std::__cxx11::basic_string is not yet supported; the struct contains an interior pointer which breaks D move semantics!");
```

Comment:
Instead, can't we just disable `opAssign` and postblit to make it un-copyable ?

Reply:
Copying is not the problem, moving is. D has no control over move semantics.